examples/metadata-examples/article-json-ld-twitter-card.amp.html

```diff
@@ -16,11 +16,19 @@
         The canonical document should also have a corresponding <link> tag
         within pointing at this AMP HTML file:
 
-          <link rel="amphtml" href="http://example.ampproject.org/article-metadata.amp.html" />
+          <link rel="amphtml" href="http://example.ampproject.org/article-json-ld-twitter-card.amp.html" />
 
         It is possible that this AMP HTML document is the canonical document
         for this article, in which case, the canonical URL should point to this
         document, and no "amphtml" link is required.
+
+        Also, please be aware that some platforms that use AMP HTML have
+        further restrictions with regards to some schema components.
+
+         For example:
+
+           * All marked-up URL's should be absolute.
```

Comment:
Can you clarify what you mean by "marked-up"?  Also, "URL's" should just be "URLs".  Can this comment simply be "All URLs should be absolute."